variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "environment" {
```

Comment:
Maybe put validations on possible values?

Reply:
The possible values would be subjective to consumer's environment names, but could add a regex value to ensure it's one word. Would that suffice?